Cargo.toml

```diff
@@ -27,6 +27,11 @@ members = [
 	"core-primitives",
 	"erasure-coding",
 	"primitives",
+	"polkadot-runtime-constants",
+	"kusama-runtime-constants",
+	"rococo-runtime-constants",
+	"test-runtime-constants",
+	"westend-runtime-constants",
```

Comment:
They should go into the runtime folder under `runtime/westend-constants` etc.

Reply:
They are already in their respective runtime folders as dependencies, how do you mean?

Reply:
He means the location in the directory structure. You put them as top-level folders, when they should be subdirectories of the runtime crates that depend on them. (so the location of the `Cargo.toml` should IMO `runtime/westend/constants/Cargo.toml`)

Reply:
@apopiak & @bkchr does that solve it?